.github/workflows/codeql.yml

```diff
@@ -29,6 +29,7 @@ jobs:
     # Consider using larger runners for possible analysis time improvements.
     runs-on: ${{ (matrix.language == 'swift' && 'macos-latest') || 'ubuntu-latest' }}
```

Comment:
We could clean this up, too, if we use an actual matrix.

And the default comments from the template are not relevant for us, so they could go away.

Reply:
Actually... This is codeQL scanning and we do not have OS-specific dependencies.

We could do it on just one runner for this workflow unless anyone has any thoughts otherwise.

Reply:
Out of curiosity, what runner would in that case take your preference? The Ubuntu one?

Reply:
May as well be that one. Likely to have the lowest total running time cost.

Reply:
Ah yes clear!